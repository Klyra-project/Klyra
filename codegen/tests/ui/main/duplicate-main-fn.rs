#[klyra_codegen::main]
async fn main() -> KlyraRocket {}
