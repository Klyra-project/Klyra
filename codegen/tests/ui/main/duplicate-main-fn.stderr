error: klyra_runtime::main functions cannot be named `main`
 --> tests/ui/main/duplicate-main-fn.rs:2:10
  |
2 | async fn main() -> KlyraRocket {}
  |          ^^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/ui/main/duplicate-main-fn.rs:2:36
  |
2 | async fn main() -> KlyraRocket {}
  |                                    ^ consider adding a `main` function to `$DIR/tests/ui/main/duplicate-main-fn.rs`
